pwndbg/dbg/lldb/__init__.py

```diff
@@ -1378,7 +1352,7 @@ def handler(
     @override
     def is_linux(self) -> bool:
         # LLDB will at most tell us if this is a SysV ABI process.
-        return self.target.GetABIName().startswith("sysv")
+        return self.target.GetABIName().lower().startswith("sysv")
```

Comment:
Please add the whole `GetABIName()` result in a docstring for this function - it will be helpful if this ever changes or if we ever need to add another case for this etc.

Also: do we really have to do `.lower()`? Do we think the letter sizes may ever be different?

Reply:
```
In [6]: inf.target.GetABIName()
Out[6]: 'SysV-arm64'

In [6]: inf.target.GetABIName()
Out[6]: 'ABIMacOSX_arm64'
```